src/Relation/Binary/TypeClasses/Examples.agda

```diff
@@ -0,0 +1,47 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Usage examples of typeclasses for binary relations
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Relation.Binary.TypeClasses.Examples where
```

Comment:
I think we're moving away from `Example` files and towards `README` files. I think I'd be tempted to move this to `README.TypeClasses` or `README.Relation.Binary.TypeClasses`. If you could also add an entry for `X.TypeClasses` in the `README.Design.Hierarchies` file that would be great.